source/utils/types.ts

```diff
@@ -59,10 +73,16 @@ export interface Options extends RequestOptions {
 	decompress?: boolean;
 	encoding?: BufferEncoding | null;
 	method?: Method;
+	retry?: RetryDescriptor;
```

Comment:
```suggestion
	retry?: number | RetryDescriptor;
```

https://github.com/sindresorhus/got#retry